lib/plugins/aws/Lambda/lib/deploy.js

```diff
@@ -395,6 +404,7 @@ functions.deploy_stacks = function(input, done) {
 
 module.exports = {
     deploy(cim, done) {
```

Comment:
@eetay ☝️ These changes look great!  Thank you.

Lets remove thi 👇  for now.  I'll debug and see what is wrong.  Sound good?

Thanks again.

Reply:
I think I have it: I'm guessing the problem is a conflict between the generic "--version" switch 
```
cim <cmd> [args]

Commands:
   ...
Options:
  --version  Show version number                                       [boolean]
```
and the --version of the Lambda.js plugin

Suggest to rename the --version to --lambda-version or something similar
If acceptable I'll make the proper changes